tracing/src/main/java/io/airlift/tracing/OpenTelemetryModule.java

```diff
@@ -63,6 +63,7 @@ public OpenTelemetry createOpenTelemetry(NodeInfo nodeInfo, OpenTelemetryConfig
                 .putAll(attribute(ResourceAttributes.OS_NAME, OS_NAME.value()))
                 .putAll(attribute(ResourceAttributes.OS_VERSION, OS_VERSION.value()))
                 .putAll(attribute(ResourceAttributes.HOST_ARCH, hostArch()));
+        nodeInfo.getAnnotations().forEach(attributes::put);
```

Comment:
@mattstep Should we namespace these somehow? Are the values typically used for log annotations appropriate to use in this way?

Reply:
Good call, I'd say yes, we basically do that in logging.

Reply:
They are in a nested field "annotations" : https://github.com/airlift/airlift/blob/master/log-manager/src/main/java/io/airlift/log/JsonRecord.java#L129

Reply:
Any ideas for the namespace? I think `annotations` would be too generic in this context. Also, as Matt mentioned, we already put this in a nested `annotations` field for logging, so this would stutter. Maybe `node-info`?

On the other hand, should we leave this for users to come up with meaningful prefix? This would also allow overriding some of the default attributes, like `service.name`, which is configurable in every OpenTelemetry instrumentation.

Reply:
It's a very generic feature, just about anything can be added in. Idk, we used annotations for logging as a name because it's just really generic. We probably don't want to support overriding the fields we set already.

Reply:
Looking through https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/attribute-naming.md#recommendations-for-application-developers I believe we should namespace the attributes something like `io.airlift.custom` for anything passed in via the annotations file, and the naming there might be better as `custom-attributes` or so... For any attributes we're adding in that are specific to airlift and not based on something in ResourceAttributes we should use the airlift packaging from the module it comes from as it's namespace, so anything derived from NodeInfo that isn't already a standard attribute would be `io.airlift.node`. How does that sound?

Reply:
Sounds good, I added `io.airlift.node`, PTAL and thanks!